contributing.md

```diff
@@ -202,7 +202,8 @@ The “page” modules themselves contain introductory blurbs and examples for t
 You can run the docs app locally using:
 
 ```sh
-npm run docs
+npm run build
```

Comment:
is not obvious that you need to create the build first, you may have done it before if you follow the whole guide but if you jump directly to documentation you may have missed the step